source/authentication.rst

```diff
@@ -311,21 +311,20 @@ Kerberos Authentication
 
 .. _ldap-authorization:
 
-Authentication and Authorization with Direct Binding to LDAP
+Authentication and authorization with direct binding to LDAP
 ============================================================
 
 As of version 3.6.18-5.0, |psmdb| supports |ldap-authorization|.
 
 This feature has been supported in |mongodb-e| since its version 3.4. 
   
-Note the following limitations of |ldap-authorization| in |psmdb|:
+.. note::
   
-- The `ldapTimeoutMS <https://docs.mongodb.com/manual/reference/program/mongoldap/#cmdoption-mongoldap-ldaptimeoutms>`_ parameter is ignored.
-- The `--ldapServers <https://docs.mongodb.com/manual/reference/program/mongoldap/#cmdoption-mongoldap-ldapservers>`_ option may only contain a single server (|mongodb-e| accepts a comma-separated list).
+   The `ldapTimeoutMS <https://docs.mongodb.com/manual/reference/program/mongoldap/#cmdoption-mongoldap-ldaptimeoutms>`_ parameter is ignored.
```

Comment:
ldapTimeoutMS is not ignored starting from version 3.6.18-5.0. Its value is used as timeout values when executing LDAP queries.